core/lib/workarea/ext/freedom_patches/dragonfly_job_fetch_url.rb

```diff
@@ -1,6 +1,25 @@
 module Dragonfly
   class Job
     class FetchUrl < Step
+      class ProxyError < RuntimeError
+        def initialize(url)
+          super <<~TXT
+            Error fetching '#{url}' with Dragonfly!
+
+            This URL is not allowed through the proxy, and therefore the
+            request was blocked by the Workarea Commerce Cloud firewall.
```

Comment:
We don't really use the term firewall much, better to use the conventional language.
```suggestion
            request was blocked by the Workarea Commerce Cloud proxy.
```